presto/dataset.py

```diff
@@ -65,23 +65,73 @@ def target_crop(row_d: Dict) -> int:
         # by default, we predict crop vs non crop
         return int(row_d["LANDCOVER_LABEL"] == 11)
 
+    @classmethod
+    def get_timestep_positions(cls, row_d: Dict, augment: bool = False) -> List[int]:
+        available_timesteps = int(row_d["available_timesteps"])
+        valid_position = int(row_d["valid_position"])
+
+        # force moving the center point if it is too close to the edges
+        if (valid_position < cls.NUM_TIMESTEPS // 2) or (
+            valid_position > (available_timesteps - cls.NUM_TIMESTEPS // 2)
+        ):
+            augment = True
```

Comment:
In this case we enter unconsciously in augmentation which might not be what we desire. Can't we put this logic inside `if not augment` and in that case not choose the `valid_position` as the `center_point` but rather the the point that keeps `valid_position` closest as possible to the center? Then it's always deterministic and we don't have to force going through the augmentation part.

Reply:
good point. tried to rewrite it here b2f1aa383e60eaf3bb6f308083d75a2a5c71e9eb